lib/optimizely/notification_center.rb

```diff
@@ -87,6 +87,12 @@ def remove_notification_listener(notification_id)
     end
 
     def clear_notifications(notification_type)
+      # Deprecated
+      clear_notification_listeners(notification_type)
+      @logger.log Logger::WARN, "'clear_notifications' is deprecated. Call 'clear_notification_listeners' instead."
+    end
+
+    def clear_notification_listeners(notification_type)
```

Comment:
nit. Please update docstrings according to rubocop standard. USable with yarddocs.